dali/operators/math/expressions/arithmetic_meta.h

```diff
@@ -367,6 +379,66 @@ REGISTER_BINARY_IMPL(ArithmeticOp::add, +);
 REGISTER_BINARY_IMPL(ArithmeticOp::sub, -);
 REGISTER_BINARY_IMPL(ArithmeticOp::mul, *);
 
+template <typename Backend>
+struct arithm_meta<ArithmeticOp::min, Backend> {
+  template <typename L, typename R>
+  using result_t = binary_result_t<L, R>;
+
+  template <typename L, typename R>
+  DALI_HOST_DEV static constexpr result_t<L, R> impl(L l, R r) {
+    auto l_ = static_cast<result_t<L, R>>(l);
+    auto r_ = static_cast<result_t<L, R>>(r);
+    return l_ <= r_ ? l_ : r_;
```

Comment:
```suggestion
    return l_ < r_ ? l_ : r_;
```
This would work as well, right?

Reply:
Yes, will change